sdk/azcore/arm/runtime/policy_bearer_token.go

```diff
@@ -28,12 +28,12 @@ type acquiringResourceState struct {
 	tenant string
 }
 
-// acquire acquires or updates the resource; only one
-// thread/goroutine at a time ever calls this function
-func acquire(state acquiringResourceState) (newResource azcore.AccessToken, newExpiration time.Time, err error) {
+// acquireAuxToken acquires a token from an auxiliary tenant. Only one thread/goroutine at a time ever calls this function.
+func acquireAuxToken(state acquiringResourceState) (newResource azcore.AccessToken, newExpiration time.Time, err error) {
 	tk, err := state.p.cred.GetToken(state.ctx, azpolicy.TokenRequestOptions{
-		Scopes:   state.p.scopes,
-		TenantID: state.tenant,
+		EnableCAE: true,
```

Comment:
Does this warrant a changelog entry or will it be transparent to callers?

Reply:
It will be transparent. The prior azidentity enables CAE for all clients by default so even if someone upgrades azcore only, the feature will still work even though azidentity would ignore this field.